### ISSUE TITLE
00974_query_profiler is flaky: queue is full for system log OpenTelemetrySpanLog

### ISSUE THREAD

**Issue by tavplubix** [ClickHouse/ClickHouse#38402]
https://s3.amazonaws.com/clickhouse-test-reports/38265/59a14d2a509a45de46066bfc8e0a9ad3327d35a7/stateless_tests__release__s3_storage__actions_.html

```2022-06-24 13:48:56 00974_query_profiler:                                                   [ FAIL ] 7.79 sec. - having stderror: 
2022-06-24 13:48:56 [f81d5fb03293] 2022.06.24 13:48:53.155086 [ 41872 ] {7a8d82e9-236d-402f-aeca-4bf0fc6ce083} <Error> SystemLog (system.opentelemetry_span_log): Queue is full for system log 'DB::OpenTelemetrySpanLog' at 1539015
2022-06-24 13:48:56 
2022-06-24 13:48:56 stdout:
2022-06-24 13:48:56 0	0
2022-06-24 13:48:56 1
2022-06-24 13:48:56 10000000000	0
2022-06-24 13:48:56 1
2022-06-24 13:48:56 
```

See also https://github.com/ClickHouse/ClickHouse/pull/37837#discussion_r897812019

**Comment by tavplubix** [comment]
https://s3.amazonaws.com/clickhouse-test-reports/0/1d661d0badaf823e1d4eb3d81eefaae5988b561f/stateless_tests__release__actions_.html

**Comment by tavplubix** [comment]
https://s3.amazonaws.com/clickhouse-test-reports/38487/4f54abf67fec04c3f3e34d6b8b4c5e57118a0a96/stateless_tests__release__s3_storage__actions_.html

**Comment by tavplubix** [comment]
Fixed in #39170
